types/ckeditor__ckeditor5-engine/src/model/node.d.ts

```diff
@@ -27,7 +27,7 @@ export default class Node {
 
     constructor(attrs?: Parameters<typeof toMap>[0]);
     getAncestors(options?: { includeSelf?: boolean | undefined; parentFirst?: boolean | undefined }): Array<Element | DocumentFragment>;
-    getAttribute(key: string): string | undefined;
+    getAttribute(key: string): string | number | boolean | undefined;
```

Comment:
are those really returning something different compared to native implementation (a string or a null)

Reply:
Yes, cause attributes for the model are saved in a `Map`.

![Screenshot from 2022-03-19 20-10-08](https://user-images.githubusercontent.com/735314/159141617-6ee78644-ba46-44c0-a312-28b7b134b444.png)
![Screenshot from 2022-03-19 20-16-15](https://user-images.githubusercontent.com/735314/159141619-2c704fe0-e31d-45d4-878a-e5b60cde9771.png)

Reply:
txh!